app/controllers/Reconcile.java

```diff
@@ -487,7 +490,8 @@ private String propQuery(Entry<String, JsonNode> entry) {
 	}
 
 	private String clean(String queryString) {
-		return queryString.replaceAll("[:+\\-=<>(){}\\[\\]^]", "");
+		return queryString.startsWith("http") || queryString.contains("-") ? "\"" + queryString + "\""
+				: queryString.replaceAll("[:+\\=<>(){}\\[\\]^]", "");
```

Comment:
This may result in an unexpected behaviour, e.g. a query like  `A+=<>(){}[]^` would be changed to `A` while the same query plus a `-`  , like e.g. `A+-=<>(){}[]^`, would result in `"A+-=<>(){}[]^"`.

For consistency either escaping everytime such characters appear or replacing also the `-` would be good.

Reply:
Updated as discussed offline.